core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/jndi/JndiDirectoryDAO.scala

```diff
@@ -238,24 +282,24 @@ trait JndiDirectoryDAO extends DirectorySubjectNameSupport with JndiSupport {
     }
 
     def updateSynchronizedDate(rawlsGroupRef: RawlsGroupRef): ReadWriteAction[Unit] = withContext { ctx =>
-      ctx.modifyAttributes(groupDn(rawlsGroupRef.groupName), DirContext.REPLACE_ATTRIBUTE, new BasicAttributes(Attr.groupSynchronizedTimestamp, new Timestamp(System.currentTimeMillis()), true))
+      ctx.modifyAttributes(groupDn(rawlsGroupRef.groupName), DirContext.REPLACE_ATTRIBUTE, new BasicAttributes(Attr.groupSynchronizedTimestamp, dateFormat.format(new Date()), true))
```

Comment:
`java.text.SimpleDateFormat` is not thread-safe. I suspect that this usage is problematic but I'm not 100% sure.